### DISCUSSION TITLE
Honeywell T6 Thermostat (TH6320ZW2007) - Fan mode text is wrong

### DISCUSSION THREAD

**Discussion by zuccheroch** [zwave-js/zwave-js#7442] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

Functionality is fine, the issue is with the text that is displayed.

Thermostat supports three fan modes that are reported in Fan mode control (commandclass 68):
0 -> Auto
1 -> On
6 -> Circulation

The driver, however, reports the following text which gets used in Home Assistant
0 -> Auto low
1 - > Low
6 -> Circulation

Can we get the driver updated to report the same text that the Thermostat uses? (Auto / On / Circulation)

Here is the relevant debug information:
"commandClass": 68,
      "commandClassName": "Thermostat Fan Mode",
      "endpoint": 0,
      "property": "mode",
      "propertyName": "mode",
      "type": "number",
      "readable": true,
      "writeable": true,
      "label": "Thermostat fan mode",
      "stateless": false,
      "commandClassVersion": 3,
      "min": 0,
      "max": 255,
      "list": true,
      "states": [
        {
          "text": "Auto low",
          "value": 0
        },
        {
          "text": "Low",
          "value": 1
        },
        {
          "text": "Circulation",
          "value": 6
        }
      ],

### Software versions

zwave-js-ui: 9.26.0
zwave-js: 13.10.3
Home Assistant Z-Wave JS Addon: 3.17.0
Home Assistant Z-Wave Integration: Driver version: 13.10.3, Server version: 1.39.0 

### Z-Wave Controller (Stick/Dongle/...)

Zooz ZST39 LR
Firmware: 1.50 

### Device information

Manufacturer: Honeywell
Model name: TH6320ZW2007
Node ID: 25

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[z-ui_2024-11-26.log](https://github.com/user-attachments/files/17928881/z-ui_2024-11-26.log)
[zwavejs_2024-11-26.log](https://github.com/user-attachments/files/17928882/zwavejs_2024-11-26.log)

**Comment by AlCalzone** [comment, marked as answer]
> The driver, however, reports the following text which gets used in Home Assistant
> 0 -> Auto low
> 1 - > Low
> 6 -> Circulation

Those are the names used by the Z-Wave specification, which Z-Wave JS very closely aligns with.
I tend to agree that they seem a bit weird when used alone without the Medium and High modes.

But we currently don't have the capability to overwrite those labels on a per device basis, and I don't think adding it warrants the additional complexity.
